packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -464,6 +464,7 @@ function NoteEditor(props: NoteEditorProps) {
 		noteToolbarButtonInfos: props.toolbarButtonInfos,
 		plugins: props.plugins,
 		fontSize: Setting.value('style.editor.fontSize'),
+		fontFamily: Setting.value('style.editor.fontFamily'),
```

Comment:
You're passing the fontFamily as a property, but you could just get it from the theme (just like for theme.lineHeight), so could you change this please?

Reply:
In order to do so, I will have to change the `fontFamily` property in the `globalStyle` object in `theme.ts`, because it's given a static value ('Roboto').
So, should I change it and use `theme.fontFamily` or stay with`props.fontFamily`?

Reply:
Oh you're right, it should indeed come from the settings since it's dynamic. All good then!